feature/main/src/main/java/io/github/droidkaigi/confsched2023/main/MainScreen.kt

```diff
@@ -84,40 +85,45 @@ fun MainScreen(
     )
 }
 
+sealed class IconRepresentation {
```

Comment:
Implement the selectedIcon of MainScreenTab to make it work with drawable.

Reply:
👍